### DISCUSSION TITLE
When having SIGNUPS_ALLOWED to false don't show option Create an account in main-screen

### DISCUSSION THREAD

**Discussion by pluim003** [dani-garcia/vaultwarden#3312] [Ideas]
I was wondering why you can still create an account when having SIGNUPS_ALLOWED set to false. After having filled in everything you will get a notification that you can't sign up.
Or is there a good reason to show the 'Create account' on the mainscreen even if account-creation isn't active?

**Comment by BlackDex** [comment]
If you would have searched, then you probably would have found this answered a dozen of times already.
But.... No, we can not remove that button from the interface, since there is no toggle to turn it on/off at runtime.

Also, in some cases, like when not having an SMTP server enabled, and you invite someone, they need to click on the create account button to still be able to create an account, since they will not receive e mail with a link.

In the end, someone is still not able to create an account and will see an error message.

**Comment by pluim003** [comment]
Thanx, I didn't search well enough then. Sorry for inconvenience.
